test/irb/test_cmd.rb

```diff
@@ -469,6 +469,17 @@ def test_show_source_method
       assert_match(%r[/irb\/init\.rb], out)
     end
 
+    def test_show_source_method_s
+      File.write("#{@tmpdir}/bazbar.rb", "class Baz\n  def foo\n  end\nend\n\nclass Bar < Baz\n  def foo\n    super\n  end\nend\n")
+      out, err = execute_lines(
+        "irb_load '#{@tmpdir}/bazbar.rb'\n",
+        "show_source Bar#foo -s",
+      )
+      assert_empty err
+      p out
```

Comment:
I think this is a leftover from debugging?

Reply:
Yes, apologies!